src/v2/Apps/Article/Components/ArticleSectionAd.tsx

```diff
@@ -0,0 +1,56 @@
+import { FullBleed } from "@artsy/palette"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArticleAd } from "./ArticleAd"
+import { ArticleSectionAd_article } from "v2/__generated__/ArticleSectionAd_article.graphql"
+
+interface ArticleSectionAdProps {
+  article: ArticleSectionAd_article
+  i: number
+}
+
+const ArticleSectionAd: FC<ArticleSectionAdProps> = ({ article, i }) => {
+  switch (article.layout) {
+    case "FEATURE": {
+      if (i === 1) {
+        return (
+          <FullBleed bg="black5" p={1}>
+            <ArticleAd unit="Desktop_Leaderboard1" size="970x250" />
+          </FullBleed>
+        )
+      }
+
+      if (i === 6) {
```

Comment:
whats `i === 1`? `i === 6`? I imagine its some kind of positional flag but is there some way we can let our future selves know this?

Reply:
I can rename to `index` — just the index of the section. I feel like `i` is the one-letter variable we can get away with 😄

Reply:
Oh i mean the 6, 8, etc

Reply:
all said, this likely can't be improved, but just wanted to call it out in case

Reply:
oh: https://github.com/artsy/force/pull/9537/files#diff-afe1a72a48bafcf27d37e846452b047bc39a82e51a59afe9d75c6f4fb84922e7

Reply:
Yeah; maybe just leaving a comment to the README on the placement.